doc/src/tutorials/creating-packages.md

```diff
@@ -605,9 +605,21 @@ may fit your package better.
 
      * `CPLEX.jl` wraps the `CPLEX` library, which can be identified easily in a web search.
      * `MATLAB.jl` provides an interface to call the MATLAB engine from within Julia.
+
 7. Avoid naming a package closely to an existing package
      * `Websocket` is too close to `WebSockets` and can be confusing to users. Rather use a new name such as `SimpleWebsockets`.
 
+8. Avoid using a distinctive name that is already in use in a well known, unrelated project.
+     * Don't use the names `Tkinter.jl`, `TkinterGUI.jl`, etc. for a package that is unrelated
+       to the popular `tkinter` python package. Users are likely to assume a connection that
+       does not exist.
+     * It's okay to name a package `HTTP.jl` even though it is unrelated to the popular rust
+       crate `http` because in most usages the name "http" refers to the hypertext transfer
+       protocol, not to the `http` python package.
+     * It's okay to name a package `AlphaZero.jl` if it provides an implementation of
+       DeepMind's AlphaZero algorithm, even without explicit affiliation with the creators of
+       the AlphaZero algorithm (provided there's no copyright or trademark infringement etc.)
```

Comment:
It looks like AlphaZero is trademarked.  IANAL and I have no idea how that impacts things like a package, but maybe this isn't the best example?

Reply:
Good point. In fact, I would not use the example of a commercial product since this might suggest that we are encouraging people to violate trademarks. A better example would be an implementation of an academic paper that describes an algorithm - like TSVD.jl.

Reply:
Though I also don't want to recommend the package name TSVD because it's kind of marginal.